cluster/gce/trusty/configure-helper.sh

```diff
@@ -726,7 +726,7 @@ Note: This looks like a development version, which might not be present on GitHu
 If it isn't, the closest tag is at:
   https://github.com/kubernetes/kubernetes/tree/${gitref}
 "
-    gitref="${version//*+/}"
+    gitref=$(echo ${version} | sed -e 's/.*+//')
```

Comment:
Not really. It is more like a fix to a previous cherrypick. The master and release-1.3 branch trusty code diverges from release-1.2 too much. Manual work was heavily involved in recent cherrypicks to release-1.2 and I made a mistake there. So I would like to fix it in this PR